api.py

```
@@ -661,4 +794,4 @@ def server_error(e):
 
 
 def run_api():
-    app.run(host="0.0.0.0", port=settings.port, debug=False, use_reloader=False)
+    app.run(host="0.0.0.0", port=settings.port, debug=False, use_reloader=False)
```

Ruff (0.15.7)

[error] Possible binding to all interfaces

(S104)